src/xtgeo/well/_well_io.py

```diff
@@ -292,7 +293,7 @@ def import_wlogs(wlogs: dict):
         else:
             raise ValueError(f"Invalid log type found in input: {typ}")
 
-        if rec is None or isinstance(rec, dict):
+        if rec is None or isinstance(rec, Iterable):
```

Comment:
Another option could be `isinstance(rec, (dict, list))`. `Iterable` will accomplish the same thing with a bit of a broader scope (also including `set` etc) so I think this is fine.